manifests/init.pp

```diff
@@ -363,6 +366,7 @@
 
     include pulpcore::plugin::container
     include pulpcore::plugin::file
+    include pulpcore::plugin::rpm
```

Comment:
general question: shouldn't those plugin includes (at least file and rpm) be inside the respective "proxy to pulpcore" blocks? as otherwise we'd be installing the pulpcore plugin but not using it, which seems like a waste?

Reply:
The idea is that the plugin is installed. This allows you to migrate content from Pulp 2. Also the Pulp 3 content URLs are always available. The "proxy to pulpcore" blocks are about routing the old Pulp 2 URLs to Pulp 3 so clients don't have to be modified. Ideally they should, but this allows to do so at a convenient pace.